### ISSUE TITLE
Web of Thing Interest Group rechartering 2021

### ISSUE THREAD

**Issue by ashimura** [w3c/strategy#274]
New charter proposal, reviewers please take note.

# Charter Review

[Charter: Web of Things Interest Group](https://w3c.github.io/wot/charters/wot-ig-2021-proposed.html)

This is a recharter for an [Existing IG](https://www.w3.org/2019/10/wot-ig-2019.html).

Other useful links:
* [Diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2019%2F10%2Fwot-ig-2019.html&doc2=https%3A%2F%2Fw3c.github.io%2Fwot%2Fcharters%2Fwg-ig-2021-proposed.html), and any issue discussion:
* [Discussion issue on GitHub](https://github.com/w3c/wot/issues/966)

Feel free to respond to the [discussion issue 966](https://github.com/w3c/wot/issues/966) or raise new issues on the [group's issue tracker](https://github.com/w3c/wot/issues).

***Note:*** The draft Charter above is still an initial draft and needs to be updated based on the feedback from the group's [virtual F2F meeting](https://www.w3.org/WoT/IG/wiki/F2F_meeting,_June_2021#Virtual_F2F) in June.

***TBD:*** Horizontal Reviews: apply the Github label "Horizontal review requested" to request reviews for accessibility (a11y), internationalization (i18n), privacy, and security.

**Comment by mlagally** [comment]
Suggested new work areas:
- Intelligent Edge
- Cloud 2 Cloud

**Comment by samuelweiler** [comment]
There is a broken link to the graph re: relationships with other groups.

Otherwise, no comments.

**Comment by ashimura** [comment]
Thanks a lot, @samuelweiler ! I've just fixed the broken link.

**Comment by himorin** [comment]
No comment/request from i18n.

**Comment by michael-n-cooper** [comment]
No comments from APA; over to @brewerj to complete accessibility horizontal review.

**Comment by ashimura** [comment]
We've updated the draft IG Charter to add clarifications, and think it's ready from the WoTT-IG's viewpoint.

@samuelweiler @himorin and @michael-n-cooper could you please skim the [updated draft Charter](https://w3c.github.io/wot/charters/wot-ig-2021-proposed.html)?

The recent changes are described by the [diff version](https://w3c.github.io/wot/charters/diff.2021-latest.html).

Please note that we added some minor tweaks to the CSS setting for (1) the background color of the "Scope Summary" section and (2) the size of the image on the relationship between the WoT-IG and the WoT-WG.

**Comment by himorin** [comment]
no comment from i18n.

**Comment by mmccool** [comment]
It seems at this point that we are ready to proceed, and will do so.

**Comment by samuelweiler** [comment]
No objection to the changes.  Sorry for missing the request for a second look.  @brewerj, did you remove the privacy review completed label to try to get my attention?

**Comment by plehegar** [comment]
https://lists.w3.org/Archives/Member/w3c-ac-members/2022JanMar/0013.html
